hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/PersonalRankTraverser.java

```diff
@@ -120,10 +131,10 @@ public PersonalRankTraverser(HugeGraph graph, double alpha,
         return incrRanks;
     }
 
-    private Map<Id, Double> compensateSource(Id source, Map<Id, Double> incrRanks) {
-        double sourceRank = incrRanks.getOrDefault(source, 0.0);
-        sourceRank += (1 - this.alpha);
-        incrRanks.put(source, sourceRank);
+    private Map<Id, Double> compensateRoot(Id root, Map<Id, Double> incrRanks) {
+        double oldRank = incrRanks.getOrDefault(root, 0.0);
```

Comment:
oldRank => rank

also change incrRanks in line 95 to ranks

Reply:
line 95 can't rename to ranks, because it's method has a param ranks

Reply:
rename to newRanks